conseil-api-tests/src/main/scala/tech/cyptonomic/conseil/api/tests/ApiTestRun.scala

```diff
@@ -0,0 +1,289 @@
+
+  override def run(args: List[String]): IO[ExitCode] = {
+
+    val httpClient: BlazeClientBuilder[IO] = createHttpClient()
+
+    //        TESTING CONSEIL BUILD REQUESTS
+
+    println("\n\n\nTesting Info Endpoint:\n")
+    println(sendConseilRequest(
+      httpClient,
+      CONSEIL.withPath(Requests.ConseilRequests.CONSEIL_BUILD_INFO),
+      Requests.TezosValidationLists.TEZOS_INFO_VALIDATION
+    ))
+
+    println("\n\n\nTesting Platforms Endpoint\n")
+    println(sendConseilRequest(
+      httpClient,
+      CONSEIL.withPath(Requests.ConseilRequests.CONSEIL_PLATFORMS),
+      Requests.TezosValidationLists.TEZOS_PLATFORMS_VALIDATION
+    ))
+
+    //        TESTING CONSEIL TEZOS BUILD CONFIG
+
+    println("\n\n\nTesting Tezos Networks\n")
+    println(sendConseilRequest(
+      httpClient,
+      CONSEIL.withPath(Requests.TezosConfig.TEZOS_NETWORKS),
+      Requests.TezosValidationLists.TEZOS_NETWORKS_VALIDATION
+    ))
+
+    println("\n\n\nTesting Tezos Entitites\n")
+    println(sendConseilRequest(
+      httpClient,
+      CONSEIL.withPath(Requests.TezosConfig.TEZOS_ENTITIES),
+      Requests.TezosValidationLists.TEZOS_ENTITIES_VALIDATION
+    ))
+
+    //        TESTING TEZOS ENTITY ATTRIBUTES
+    testAttributes(httpClient)
+
+    testAttributeData(httpClient)
+
+    validateAttributeData(httpClient)
+
+
+
+    //        TESTING TEZOS CHAIN DATA ENDPOINTS
+
+    println("\n\n\nTesting Block Head\n")
+    println(sendConseilRequest(httpClient, CONSEIL.withPath(Requests.TezosChainRequests.TEZOS_BLOCK_HEAD)))
+
+    println("\n\n\nTesting Operation Kinds\n")
+    println(sendConseilRequest(httpClient, CONSEIL.withPath(Requests.TezosChainRequests.TEZOS_OPERATION_KINDS)))
+
+
+    IO.pure(ExitCode.Success)
+  }
+
+  /**
+   * Create the http client used to make requests
+   * @return The client object
+   */
+  def createHttpClient(): BlazeClientBuilder[IO] = {
+
+    val pool = Executors.newCachedThreadPool()
+    val clientExecution: ExecutionContext = ExecutionContext.fromExecutor(pool)
+
+    val clientBuild = BlazeClientBuilder[IO](clientExecution)
+
+    clientBuild
+  }
+
+  /**
+   * Send a conseil request with an API Key and proper headers
+   * @param client The BlazeClientBuilder client with which to make the request
+   * @param queryUrl The URI to query
+   * @return String results from the query performed
+   */
+  def sendConseilRequest(client: BlazeClientBuilder[IO], queryUrl: Uri, validationItems: List[String] = List[String]()): String = {
+
+    val conseilRequest = GET(
+      queryUrl,
+      `Content-Type`(MediaType.application.json),
+      Accept(MediaType.application.json),
+      Header("apiKey", CONSEIl_API_KEY)
+    )
+
+    val result: String = client.resource.use { client =>
+      client.expect[String](conseilRequest)
+    }.unsafeRunSync()
+
+    if(validationItems.nonEmpty) validationItems.foreach(item => {
```

Comment:
you can write `validationItems.foreach(item => { ` without using `(` - similarly in other cases below